ArchiverAccess/log_file_generator.py

```diff
@@ -16,19 +16,12 @@
 """
 Module for creating a log file
```

Comment:
I gather the purpose of this file is for testing large databases - can you please clearly document this somewhere in here?

Reply:
Created a script see https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Archive-access